charts/nautobot/templates/network-policy.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.networkPolicy.enabled }}
+apiVersion: {{ include "common.capabilities.networkPolicy.apiVersion" . }}
+kind: NetworkPolicy
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  {{- $podLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.nautobot.podLabels .Values.commonLabels ) "context" . ) }}
+  podSelector:
+    matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 6 }}
+  policyTypes:
+    - Ingress
+    - Egress
+  {{- if .Values.networkPolicy.allowAllEgress }}
+  egress:
+    - {}
+  {{- else }}
+  egress:
+    # Allow DNS resolution
+    - ports:
+        - port: 53
+          protocol: UDP
+        - port: 53
+          protocol: TCP
+    # Allow connectivity to backend services (DB, redis, etc)
+    {{- if or .Values.postgresql.enabled .Values.postgresqlha.enabled }}
+        - port: 5432
+    {{- else }}
+        - port: {{ .Values.nautobot.db.port }}
+    {{- end }}
+          protocol: TCP
+    {{- if .Values.redis.enabled }}
+        - port: 6379
+    {{- else }}
+        - port: {{ .Values.nautobot.redis.port }}
+    {{- end }}
+          protocol: TCP
+    {{- if .Values.networkPolicy.extraEgress }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.networkPolicy.extraEgress "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- end }}
+  ingress:
+    {{- if .Values.networkPolicy.allowExternalIngress}}
+    # Allow connectivity from everyone (including outside the cluster) into the Web application
+    - ports:
+        - port: 8443
+          protocol: TCP
+        - port: 8080
+          protocol: TCP
```

Comment:
How does this policy change if we enable nginx: https://github.com/nautobot/helm-charts/blob/154798a0c6eb80dbaa85d5977b17d104bbf3ce3e/charts/nautobot/templates/nautobot-deployment.yaml#L313-L320

Reply:
Nice catch! Updated it